config/webpack.dev.config.js

```diff
@@ -69,11 +71,14 @@ module.exports = Merge.smart(commonConfig, {
   devServer: {
     host: '0.0.0.0',
     port: 18011,
-    proxy: {
-      '/api': {
-        target: targetUrl,
-        pathRewrite: { '^/api': '' },
-      },
-    },
+    proxy: {},
   },
 });
+
+Object.keys(apiEndpoints).forEach((endpoint) => {
+  devConfig.devServer.proxy[apiEndpoints[endpoint]] = {
+    target: targetUrl,
+  };
```

Comment:
I actually don't think you need the `target` unless you're doing a `pathRewrite`, so an object like
```
{
    <apiEndpoint1>: targetUrl,
    <apiEndpoint2>: targetUrl,
    // ...
}
```
should be fine. You could also do this inline as part of the config itself with a `map` statement, e.g.
```
proxy: Object.keys(apiEndpoints).map(endpoint => apiEndpoints[endpoint])
```

ES2015 FTW!